app/controllers/time_regs_controller.rb

```diff
@@ -55,6 +58,7 @@ def destroy
   end
 
   def toggle_active
+    @time_regs_week.find(&:active?)&.deactivate!
```

Comment:
Also, I believe we do not need to deactivate every `active` time_regs on their behalf at this point. Let's have a separate cleanup JOB if we do need to cleanup active time_regs. I think returning an error message would just be simple and helpful, and we let the user choose what to do with their active `time_reg`. Also just to expand the area of active `time_regs` I think it wouldn't only be `@time_regs_week` but all active for the user

Reply:
Of course. You're absolutely right. :sweat_smile: